### DISCUSSION TITLE
prisma 2 no longer generates prisma.graphql

### DISCUSSION THREAD

**Discussion by mitchhankins01** [prisma/prisma#2115] [Q&A] [answered]
## Problem
prisma 1 generated "/generated/prisma.graphql" which allowed us to import types into the apollo scheme, prisma 2 no longer generates this file

## Solution
have prisma 2 generate the file as well, or is there a better way?

**Comment by notrab** [comment, marked as answer]
@mitchhankins01 Prisma 2 is no longer just a GraphQL ORM but more abstract ORM. I'd look at [Nexus](https://www.nexusjs.org/) + Prisma plugin for generating such schemas.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
